### PR TITLE
Add User Stories to DeveloperGuide.md

### LINKED ISSUE #19: Add User Stories to Developer Guide
The User stories needs to be included in the developer guide. Each user story should be assigned a priority

### PR DESCRIPTION
**Pull request by Imranr2**
Closes #19

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -268,18 +268,94 @@ _{Explain here how the data archiving feature will be implemented}_
 
 ### User stories
 
-Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
+Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to have)
+
+<table>
+  <tr>
+   <th colspan="3"><h3><strong>A gym manager</strong></h3></th>
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:275]: Change to 'As a gym manager'

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -268,18 +268,94 @@ _{Explain here how the data archiving feature will be implemented}_
...
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Filter bookings based on time and date</td>
+    <td>I know how many bookings are made and who are the clients</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>View the list of available commands</td>
+    <td>I know what the application </td>
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:355]: Change to "I can make full use of the app's features"

**Review by EuniceSim142 — COMMENTED** [review]
LGTM! Just some minor nits

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -268,18 +268,94 @@ _{Explain here how the data archiving feature will be implemented}_
...
+   <th colspan="3"><h3><strong>A gym manager</strong></h3></th>
+  </tr>
+  <tr>
+    <th><h3><strong>Priority</strong></h3></th>
+    <th><h3><strong>I can...</strong></h3></th>
+    <th><h3><strong>So that...</strong></h3></th>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>View client's details</td>
```
Comment (zognin) [inline docs/DeveloperGuide.md:284]: Could you put "a" in front of all "client's"?
```suggestion
    <td>View a client's details</td>
```

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -268,18 +268,94 @@ _{Explain here how the data archiving feature will be implemented}_
...
+    <td>I can confirm if they have a booking before letting them enter</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Delete clients</td>
+    <td>I will not clutter my app</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Make booking appointments for client</td>
```
Comment (zognin) [inline docs/DeveloperGuide.md:324]:
```suggestion
    <td>Make booking appointments for clients</td>
```

**Review by zognin — APPROVED** [review]
LGTM! Please link the issue to your PR description in future.

**Pull request merged**